app/code/Magento/Fedex/Test/Unit/Plugin/Block/Tracking/PopupDeliveryDateTest.php

```diff
@@ -0,0 +1,90 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Fedex\Test\Unit\Plugin\Block\Tracking;
+
+use Magento\Fedex\Plugin\Block\Tracking\PopupDeliveryDate;
+use Magento\Framework\TestFramework\Unit\Helper\ObjectManager;
+use Magento\Shipping\Block\Tracking\Popup;
+use Magento\Shipping\Model\Tracking\Result\Status;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Unit Test for @see \Magento\Fedex\Plugin\Block\Tracking\PopupDeliveryDate
+ */
+class PopupDeliveryDateTest extends TestCase
+{
+    /**
+     * @var MockObject|PopupDeliveryDate
+     */
+    private $plugin;
+
+    /**
+     * @inheritDoc
+     */
+    protected function setUp()
+    {
+        $objectManagerHelper = new ObjectManager($this);
+        $this->plugin = $objectManagerHelper->getObject(PopupDeliveryDate::class);
+    }
+
+    /**
+     * Test the method with Fedex carrier
+     */
+    public function testAfterFormatDeliveryDateTimeWithFedexCarrier()
+    {
+        /** @var Status|MockObject $trackingStatusMock */
+        $trackingStatusMock = $this->getMockBuilder(Status::class)
+            ->disableOriginalConstructor()
+            ->setMethods(['getCarrier'])
+            ->getMock();
+        $trackingStatusMock->expects($this::once())
+            ->method('getCarrier')
+            ->willReturn(\Magento\Fedex\Model\Carrier::CODE);
```

Comment:
Could you please `use Magento\Fedex\Model\Carrier;`  and change this line to `->willReturn(Carrier::CODE);`

Reply:
Thank you for the notice, done.

Reply:
Thank you !